src/generate-assert-identifier.ts

```diff
@@ -49,15 +56,19 @@ const addImport = (scope: Scope, t: typeof BabelTypes, source: string) => {
 
 export default (
   t: typeof BabelTypes,
-  { autoImport: importSource }: InternalConfig,
+  { autoImport: importSource, assertFunctionName }: InternalConfig,
 ) => (scope: Scope) => {
   if (importSource) {
     const name = findExistingImportFromSource(scope, t, importSource);
     if (name) {
       return t.identifier(name);
     }
 
-    return addImport(scope, t, importSource);
+    return addImport(scope, t, importSource, assertFunctionName);
+  }
+
+  if (assertFunctionName) {
```

Comment:
Could we get rid of the if statement and make the whole thing return t.identifier(assertName || ASSERT_IDENTIFIER_NAME)?

Reply:
done